README.md

```diff
@@ -26,12 +37,42 @@ Current research is targeted to explore and analyze data from WorldSkills Russia
+
+_alternative hypothesis (h1):  
+The repeated participation in WorldSkills competitions significantly improves a competitor's average score._
+
+2. Effect of repeated participation of a compatriot expert on competitor results
+
+_null hypothesis (h0):  
+The repeated participation of a compatriot expert does not significantly improve his/her competitor's (average) results in the competitions of WorldSkills Russia._  
+
+_alternative hypothesis (h1):  
+The repeated participation of a compatriot expert significantly improves his/her compatriot competitor's average results in the competitions of WorldSkills Russia._
+
 ## Operationalization of the research questions
 <!-- Describe the data that will be used and how the questions will be answered on the basis of this data. The data analysis itself is not yet described here. So do tell ‘Data file [X] comes from [Y] and can answer the posed questions because [Z]’, but do not yet describe the data itself -->
 
 Each competition event uses the Competition Information System (CIS) to record competition results. A separate instance of CIS is used for each event, so all event results are stored in a separate database instance. In order to have an overview of all competition events, WorldSkills Russia has developed its own aggregation system called Electronic System for Internet Monitoring (eSIM).
 
-The raw data used for this research was kindly provided by the autonomous non-profit organisation "Agency for the Development of Professional Excellence (WorldSkills Russia)" and is essentially a dump from the eSIM database, specifically from the competition results table. Therefore, the observation unit is a skills competition result for a given competitor for a given competition event.
+The raw data used for this research was kindly provided by the autonomous non-profit organization "Agency for the Development of Professional Excellence (WorldSkills Russia)" and is essentially a dump from the eSIM database, specifically from the competition results table. Therefore, the observation unit is a skills competition result for a given competitor for a given competition event.
+
+The following steps will make sure to operationalize the data so that the research questions that are formulated will be answered:
+- Data preparation  
+- Data cleaning
+- Calculate initial summary statistics  
+- Calculate the average scores of competitors  
+  --> for each competitor, calculate the average score across competitions they have participated in  
+  --> group the data by competitors and then calculate the mean of the marks for each group  
+- Asses the impact of repeated participation
+- Examine the influence of compatriot experts  
+  --> analyze whether the presence of a compatriot expert has a significant impact on the competitor's average scores.  
+  --> group competitors by the presence or absence of compatriot experts and compare their average scores  
```

Comment:
If we will disregard demonstration exams, we will have only competition results where competitors always have a compatriot expert (even if it's not reflected in the actual data frame). 

If we would like to group something the best option is to group by regions (in both questions) - to see e. g. which how repeated participation (for both competitors and experts) gives different scores for different regions.

Reply:
Agree, I will edit this as well

Reply:
One question though, if we group by region for example and see how repeated participation give different scores for different regions, are we not comparing scores from other competitions? I'm asking 
because we can't compare these as the formula for calculating the mark was dissimilar each time?

Reply:
The idea is that we are comparing not the actual results but whether repetitive participation gives an improvement in score (either for a competitor or for an expert). I've added the rest of the script with general plots which shows that this is not true (for both hypotheses) in general, but maybe it can be true for a some regions.

Reply:
Ok great, will check these tonight.